zgw_consumers/admin.py

```diff
@@ -33,5 +34,21 @@ class ListZaaktypenMixin:
 
     def formfield_for_dbfield(self, db_field, request, **kwargs):
         if db_field.name in self.zaaktype_fields:
-            return get_zaaktype_field(db_field, request, **kwargs)
+            try:
+                return get_zaaktype_field(db_field, request, **kwargs)
+            except ClientError as ex:
```

Comment:
it doesn't seem right to me that the `try` is here and on failure it returns `None`, resulting in the form field not being present at all in the admin anymore.

Instead, I would expect this `try` behaviour around the actual `get_zaaktypen` call inside `get_zaaktype_field`, as close to the actual network call as possible. You can then still return the admin field, but make the `choices` empty so that no available options are shown, but the label etc. is still visible.